sympy/stats/crv_types.py

```diff
@@ -3550,6 +3552,65 @@ def Reciprocal(name, a, b):
     """
     return rv(name, ReciprocalDistribution, (a, b))
 
+#-------------------------------------------------------------------------------
+# Rician distribution --------------------------------------------------------
+
+class RicianDistribution(SingleContinuousDistribution):
+    _argnames = ('alpha', 'beta')
+
+    @property
+    def set(self):
+        return Interval(0, oo)
+
+    @staticmethod
+    def check(alpha, beta):
+        _value_check(alpha > 0, "Shape parameter alpha must be positive.")
+        _value_check(beta > 0, "Shape parameter beta must be positive.")
```

Comment:
`alpha` and `beta` can be equal to zero.

Reply:
Also, the error notes are wrong as `alpha` and `beta` aren't shape/scale parameter. See [https://en.wikipedia.org/wiki/Rice_distribution#Characterization](https://en.wikipedia.org/wiki/Rice_distribution#Characterization)

Reply:
This was stated in the [second reference](https://reference.wolfram.com/language/ref/RiceDistribution.html) from Wolfram. 

> RiceDistribution[m,α,β] represents a continuous statistical distribution supported on the interval  and parametrized by positive real numbers α and β (called "shape parameters") and by a non-negative real number m that together determine the overall behavior of its probability density function (PDF).

But it seems that from the first reference as you pointed out [here](https://en.wikipedia.org/wiki/Rice_distribution#Characterization)

> alpha = distance between the reference point and the center of the bivariate distribution,
beta = spread

I went with Wolfram since it seems most users would be quite familiar with it. But it seems the Wikipedia reference is more mathematically accurate. Should I make this change or let it be?